fix: updating links across the React docs (#6054)
Unleash/unleash: website/docs/feature-flag-tutorials/react/examples.md

Unleash has a few ways to help manage canary deployments for React apps at scale:

> * Using a [gradual rollout](/reference/activation-strategies#gradual-rollout) (which we [implemented above](#gradual-rollouts-for-react-apps)) would be a simple use case but would reduce the amount of control you have over who gets the new feature.

Reviewer: I suggest avoiding the word "above" here because blind users don't necessarily have any directional concept of text. Microsoft mention it in their writing guide [in reference to screen content](https://learn.microsoft.com/en-us/style-guide/accessibility/writing-all-abilities#use-content-structure-and-location-to-communicate). I feel like I've heard the same about text in general, but I'm not sure. Happy to be wrong on this.

```suggestion
* Using a [gradual rollout](/reference/activation-strategies#gradual-rollout) (which we [implemented in a previous section](#gradual-rollouts-for-react-apps)) would be a simple use case but would reduce the amount of control you have over who gets the new feature.
```

In the context of our [React tutorial](/feature-flag-tutorials/react), we have a notifications badge feature that displays in the top navigation menu. To implement feature flag variants for an A/B Test in React, we will set up a variant in the feature flag and use an announcement icon from Material UI to render a different version.

In Unleash, navigate to the feature flag’s Variants tab and add a variant.

> ![We point to the button in Unleash to add a variant](/img/react-ex-add-variant.png)

Reviewer: Adding the label here is probably a good idea.

```suggestion
![We point to the button in Unleash to add a variant, labeled "add variant"](/img/react-ex-add-variant.png)
```

In the Variants form, add two variants that will reference the different icons we will render in our app. Name them ‘notificationsIcon’ and ‘announcementsIcon’. 

Note: We won’t use any particular payload from these variants other than their default returned objects. For this example, we can keep the variant at 50% weight for each variant, meaning there is a 50% chance that a user will see one icon versus the other. You can adjust the percentages based on your needs, such as making one icon a majority of users would see by increasing its weight percentage. Learn more about [feature flag variant properties](/reference/feature-toggle-variants).

![The form allows you to configure your variant.](/img/react-ex-variant-form.png)

Once you click 'Save variants' at the bottom of the form, your view will display the list of variants in the development environment with their respective metadata.

> ![Your feature flag view shows a variant has been added.](/img/react-ex-variant-added.png)

Reviewer: Maybe add a "that"? Optional, though. I'll leave it to you.

```suggestion
![Your feature flag view shows that a variant has been added.](/img/react-ex-variant-added.png)
```

### Enable feature flag impression data

At the flag level in Unleash, navigate to the Settings view.

> ![Image of editing flag Settings in Unleash](/img/react-ex-edit-settings.png)

Reviewer: Screen readers already announce that it's an image before reading the alt text, so you don't need to specify "image of" in alt text. Here is a nice [write-up about alt texts](https://axesslab.com/alt-texts/) by a vision-impaired developer who uses a screen-reader in his day-to-day. It mentions avoiding starting the text with "image of", plus many more things (such as finishing with full stops).

```suggestion
![Editing feature flag settings in Unleash.](/img/react-ex-edit-settings.png)
```

In the Settings view, click on the edit button. This will take us to the ‘Edit Feature toggle’ form.

> ![Image of enabling impression data for a feature flag](/img/react-ex-enable-impression-data.png)

Reviewer: ```suggestion
![Enabling impression data for a feature flag.](/img/react-ex-enable-impression-data.png)
```

### Application Metrics & Monitoring

Under the Metrics tab, you can see the general activity of the [Cypress Real World App from our React tutorial](/feature-flag-tutorials/react) in the development environment over different periods of time. If the app had a production environment enabled, we would also be able to view the amount of exposure and requests the app is receiving over time.

> ![Image of Metrics graph in Unleash](/img/react-ex-metrics.png)

Reviewer: ```suggestion
![The metrics graph in Unleash, showing exposure rate over the last 24 hours.](/img/react-ex-metrics.png)
```

Unleash provides the data to log any change that has happened over time, at the flag level from a global level. In conjunction with Unleash, tools like [Splunk](https://www.splunk.com/) can help you combine logs and run advanced queries against them. Logs are useful for downstream data warehouses or data lakes.

In our React tutorial application, we can view Event logs to monitor the changes to flag strategies and statuses we have made throughout our examples, such as:

* When the `newNotificationsBadge` flag was created
* How the gradual rollout strategy was configured
* When and how the variants were created and configured

> ![View the event logs for a flag](/img/react-ex-event-log.png)

Reviewer: ```suggestion
![Feature flag event log. The flag's variant's have been updated.](/img/react-ex-event-log.png)
```

In Unleash Enterprise, we have a change request feature in your project settings to manage your feature flag approvals. When someone working in a project needs to update the status of a flag or strategy, they can follow our approval workflow to ensure that one or more team members review the change request.

> ![Image of Project settings in Unleash](/img/react-ex-project-settings.png)

Reviewer: ```suggestion
![Change request configuration page under project settings.](/img/react-ex-project-settings.png)
```

We have several statuses that indicate the stages a feature flag could be in as it progresses through the workflow. This facilitates collaboration on teams while reducing risk in environments like production. For larger scale change management, you can ensure the correct updates are made while having full visibility into who made the request for change and when.

> ![Image of change requests in Unleash](/img/react-ex-change-requests.png)

Reviewer: ```suggestion
![Change request waiting for approval. Disables flag CR-toggle-3.](/img/react-ex-change-requests.png)
```
